infra/binder/src/main/java/org/apache/shardingsphere/infra/binder/statement/dml/SelectStatementContext.java

```diff
@@ -294,8 +296,20 @@ private String getOrderItemText(final TextOrderByItemSegment orderByItemSegment)
      *
      * @return group by and order by sequence is same or not
      */
-    public boolean isSameGroupByAndOrderByItems() {
-        return !groupByContext.getItems().isEmpty() && groupByContext.getItems().equals(orderByContext.getItems());
+    public boolean isGroupByStartsWithOrderByItems() {
```

Comment:
Hi @yangsen00, can you add e2e test for this logic change?

Reply:
Of course.but Can you give a Can you give an example？

Reply:
@yangsen00 You may refer to https://github.com/apache/shardingsphere/tree/618548ed8bfc07771bc659ff82b68fe6e9a5d5fd/test/e2e/sql/src/test/resources/cases/dql